src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -653,5 +653,10 @@ public override IFileSystemObject GetFileSystemInfo(string fullPath, bool asDire
                 (IFileSystemObject)new DirectoryInfo(fullPath, null) :
                 (IFileSystemObject)new FileInfo(fullPath, null);
         }
+
+        public override string[] GetLogicalDrives()
+        {
+            return DriveInfoInternal.GetLogicalDrives();
```

Comment:
I need to use the same exact code in System.Environment. I know it's one line, but it still would be duplicated- which I don't want to do to facilitate keeping them in sync. (Say if Windows decides to allow this API in the Store- which isn't unlikely.)